src/Tasks/AlgoliaIndexTask.php

```diff
@@ -249,7 +260,12 @@ private function syncChanges($index) {
     }
 
     /**
-     * All pages that have been removed will
+     * remove all AlgoliaObjects of wich the page have been removed or ShowInSearch in the CMS has been set to false
```

Comment:
Remove all AlgoliaObjects of **which** the page **has** been removed, or ShowInSearch in the CMS has been set to false.